src/main/java/frc/robot/Constants.java

```diff
@@ -16,4 +16,8 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+    public static final int RLM = 1;
```

Comment:
Please rename constants with more descriptive names, perhaps in `CONSTANT_CASE`: 
`RFM` --> `RIGHT_LEADER_ID`
These are the CAN ID's of the components, and should be named accordingly.

Reply:
This is something I plan to go over at the next meeting, I realized that I haven't covered enough.